### ISSUE TITLE
Cannot login, access token mapping broken

### ISSUE THREAD

**Issue by pulkitsinghal** [strongloop-community/loopback-connector-elastic-search#64]
Log shared by @aquid:
```
Unhandled error for request GET /api/MyModel/profile?id=AVlqFgCzgZDS4o0PX3Uf: AssertionError: token.created must be a valid Date
    at ModelConstructor.AccessToken.validate (/Users/xxx/yyy/node_modules/loopback/common/models/access-token.js:137:7)
```

**Comment by pulkitsinghal** [comment]
I don't understand how it was working until now.
We can borrow the same strategy from other connectors which have converted from `string` to `date` and fix this.

1. https://github.com/janober/loopback-connector-couchbase-expert/issues/1
2. https://github.com/janober/loopback-connector-couchbase-expert/commit/535c4581ac808ce9736d449c0dd963d2340cdc30#diff-fe2b0cd5c2de811c9832f908ab354ff4R514
3. https://github.com/janober/loopback-connector-couchbase-expert/commit/535c4581ac808ce9736d449c0dd963d2340cdc30#diff-fe2b0cd5c2de811c9832f908ab354ff4R537

**Comment by pulkitsinghal** [comment]
released in `v1.3.2`
